src/types/maybe.ts

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { Ref, UnwrapRef } from 'vue'
 
-export type MaybePromise<T> = T | Promise<T>
-export type MaybeRef<T> = T | Ref<T>
-export type MaybeUnwrapRef<T> = T | UnwrapRef<T>
-export type MaybeArray<T> = T | T[]
\ No newline at end of file
+export type MaybePromise<T = any> = T | Promise<T>
```

Comment:
Is `unknown` incorrect here?

Reply:
Great question. I wanted to be able to use these types without adding a generic every time. And in `MaybePromise<T>` `T` is unknown but it requires you pass it in because there is no default. Which is why I added the `any`. But taking another look at it I'm realizing I can set the default to `unknown` which seems much safer. 

Updated!